### PR TITLE
fix hash in language toggle

### PR DESCRIPTION
**Pull request by sebastien-comeau**
### Description

List of proposed changes:

- fix hash in language toggle

### Additional Notes

Omit the location hash in the header's language toggle link.

### CODE REVIEW

File: frontend/src/components/Header.tsx [hunk 1]
```diff
@@ -59,7 +59,11 @@ const Header: FC<HeaderProps> = ({ gocLink, skipToMainText }) => {
                     ? 'Government of Canada'
                     : 'Gouvernement du Canada'
                 }
-                src={locale === 'en' ? '/assets/sig-blk-en.svg' : '/assets/sig-blk-fr.svg'}
+                src={
+                  locale === 'en'
+                    ? '/assets/sig-blk-en.svg'
+                    : '/assets/sig-blk-fr.svg'
+                }
```
Comment (gregory-j-baker) [inline frontend/src/components/Header.tsx:66]:
```suggestion
                src={/assets/sig-blk-${locale}.svg}
```

**Review by gregory-j-baker — CHANGES_REQUESTED** [review]
Does this de-complicate the code?

File: frontend/src/components/Header.tsx (hunk 1 above)
Comment (sebastien-comeau) [reply frontend/src/components/Header.tsx:66]: What do you mean?

Comment (gregory-j-baker) [reply frontend/src/components/Header.tsx:66]: :question:

Comment (gregory-j-baker) [reply frontend/src/components/Header.tsx:66]: :point_up:

**Review by gregory-j-baker — APPROVED** [review]

**Pull request merged**